### PR TITLE
set default resolution for camera

### PR DESCRIPTION
**Pull request by arun3528**
# Pull Request Template
Make sure the resolution is 720p for video all the time 
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js [hunk 1]
```diff
@@ -21,7 +28,21 @@ MeetingsUtil.extractDestination = (destination, type) => {
     dest = temp[temp.length - 1];
   }
 
-  return dest;
+  if (isUUID.anyNonNil(dest)) {
```
Comment (taymoork2) [inline packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: replace with `/^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i.test(dest)`

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js [hunk 2]
```diff
@@ -21,7 +28,21 @@ MeetingsUtil.extractDestination = (destination, type) => {
...
+  // assume it's a convo ID
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: I would replace this with a more comprehensive header at the top of the `if` statement. 

```
// If the destination is a UUID, assume it is an internal conversation ID.
// Next, assume it is a valid Hydra room ID. 
// If decoding fails, it is a SIP or other destination.
```

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js [hunk 3]
```diff
@@ -21,7 +28,21 @@ MeetingsUtil.extractDestination = (destination, type) => {
...
+    return dest;
+  }
+
+
+  // desconstructHydraId(dest);
+  // id - UUID - conversation ID
+  // type - 'ROOM'
+  try {
+    return deconstructHydraId(dest).id || dest;
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: This should probably be more specific. What's the actual error when `deconstructHydraId` fails? 

The `||` may be sufficient; but remove the try/catch in that case.

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js [hunk 4]
```diff
@@ -21,7 +28,21 @@ MeetingsUtil.extractDestination = (destination, type) => {
     dest = temp[temp.length - 1];
```
[the remark this answers is not in the document]
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: Thats for the locus id parser

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js (hunk 2 above)
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: done

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js (hunk 3 above)
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: done

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js (hunk 1 above)
Comment (taymoork2) [reply packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: you can make the regex a variable that describes what it does or just leave a comment describing what it's testing for

File: package.json [hunk 5]
```diff
@@ -175,6 +175,7 @@
     "husky": "^0.14.3",
     "in-publish": "^2.0.0",
     "is-function": "^1.0.1",
+    "is-uuid": "^1.0.2",
```
[the remark this answers is not in the document]
Comment (mattnorris) [reply package.json]: I pointed this out, so my fault, @arun3528 😉 . I agree, @taymoork2 , we should be very choosy about what to bring in.

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js [hunk 6]
```diff
@@ -21,7 +28,16 @@ MeetingsUtil.extractDestination = (destination, type) => {
...
   }
 
-  return dest;
+  // If the destination is a UUID, assume it is an internal conversation ID.
+  if (isUUID.anyNonNil(dest)) {
+  // assume it's a convo ID
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: Remove (you explain this at the top).

File: packages/node_modules/@webex/plugin-meetings/src/config.js [hunk 7]
```diff
@@ -31,6 +31,12 @@ export default {
     logging: {
       enable: true,
       verboseEvents: true
+    },
+    resolution: {
+      maxWidth: 1280,
+      maxHeight: 720,
+      idleWidth: 1280,
+      idleHeight: 720
```
Comment (CapOM) [inline packages/node_modules/@webex/plugin-meetings/src/config.js]: ideal or idle ?

Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/config.js]: +1 this should be ideal

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js (hunk 1 above)
Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: store the regex in the constants file, why are these uuid formats not in the isUUID lib?

**Comment by arun3528** [comment]
Thanks @CapOM  @mattnorris  Fixed the review comments

File: packages/node_modules/@webex/plugin-meetings/src/meetings/util.js (hunk 1 above)
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: Its not there in the UUId lib i couldn't find any . Taymore let me check it . There is some 5 version of uuid i guess

Comment (taymoork2) [reply packages/node_modules/@webex/plugin-meetings/src/meetings/util.js]: https://github.com/afram/is-uuid/blob/master/lib/is-uuid.js#L35

File: package-lock.json [hunk 8]
```diff
@@ -6781,8 +6781,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
+          "bundled": true
```
Comment (mattnorris) [inline package-lock.json]: Was there a package change? If not, please remove this file from the PR. Generally this should only update when you update a package.

**Pull request merged**